WordPress/Classes/ViewRelated/Post/WPLegacyEditPostViewController.m

```diff
@@ -1071,7 +1071,18 @@ - (void)assetsPickerController:(CTAssetsPickerController *)picker didFinishPicki
 - (BOOL)assetsPickerController:(CTAssetsPickerController *)picker shouldSelectAsset:(ALAsset *)asset
 {
     if ([asset valueForProperty:ALAssetPropertyType] == ALAssetTypePhoto) {
-        return picker.selectedAssets.count < MaximumNumberOfPictures;
+        // If the image is from a shared photo stream it may not be available locally to be used
+        if (!asset.defaultRepresentation) {
+            [WPError showAlertWithTitle:NSLocalizedString(@"Cannot select this image", @"The title for an alert that says the image the user selected isn't available.")
+                                message:NSLocalizedString(@"This image is currently not available locally to be used in your post. Please try again later.", @"User information explaining that the image is not available locally. This is normally related to share photo stream images.")];
```

Comment:
as suggested in [previous pr](https://github.com/wordpress-mobile/WordPress-iOS/pull/3028#discussion_r22801583), pls standardize to: 

@"This Photo Stream image cannot be added to your WordPress. Try saving to your Camera Roll before uploading."

> OK